doc/lua_api.txt

```diff
@@ -6092,6 +6092,9 @@ object you are working with still exists.
           (default: `false`)
         * `new_move`: use new move/sneak code. When `false` the exact old code
           is used for the specific old sneak behaviour (default: `true`)
+        * `new_jump`: use the new jump/bouncy jump code. When `false` the exact
+          old code is used for jumping and bouncy jumping behavior.
+          (default: `true`)
```

Comment:
shouldn't this be false by default?

Reply:
shouldn't this rather be a float for jump "floatiness"?

Reply:
That sounds like too much configurability that most people won't need.

Reply:
I'm actually kinda weirded out hy such a thing would be optional, that's one of the first annoying things I found about Minetest and it's the way jumping works.

Reply:
Updated to make it false by default